assets/js/components/KeyMetrics/MetricTileText.js

```diff
@@ -0,0 +1,85 @@
+/**
+ * MetricTileText component.
+ *
+ * Site Kit by Google, Copyright 2023 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * External dependencies
+ */
+import PropTypes from 'prop-types';
+
+/**
+ * Internal dependencies
+ */
+import ChangeBadge from '../ChangeBadge';
+import PreviewBlock from '../PreviewBlock';
+import { expandNumFmtOptions } from '../../util';
+
+export default function MetricTileText( {
+	Widget,
+	loading,
+	title,
+	metricValue,
+	metricValueFormat,
+	subText,
+	previousValue,
+	currentValue,
+} ) {
+	if ( loading ) {
+		return (
+			<Widget noPadding>
+				<PreviewBlock width="100%" height="142px" padding />
+			</Widget>
+		);
+	}
```

Comment:
Let's show the loading state beneath the title as we do it in the numeric tile: https://github.com/google/site-kit-wp/blob/c1024b3e156d370c167067fc58710946c4efdb44/assets/js/components/KeyMetrics/MetricTileNumeric.js#L53-L77
And in the table tile: https://github.com/google/site-kit-wp/blob/c1024b3e156d370c167067fc58710946c4efdb44/assets/js/components/KeyMetrics/MetricTileTable.js#L81-L97